packages/salesforcedx-vscode-core/test/vscode-integration/commands/templates/forceFunctionCreate.test.ts

```diff
@@ -59,18 +59,22 @@ describe('Force Function Create', () => {
 
   describe('Pull Dependencies', () => {
     let execStub: SinonStub;
+    let sandbox: SinonSandbox;
     let notificationServiceStub: SinonStub;
+    let telemetryServiceStub: SinonStub;
     let withProgressStub: SinonStub;
     const functionInfoJS = { fileName: 'myFunc1', outputdir: 'some/dir', language: 'javascript' };
     const functionInfoJava = { fileName: 'myFunc1', outputdir: 'some/dir', language: 'java' };
     let rootWorkspacePathStub: SinonStub;
 
     beforeEach(() => {
-      execStub = stub(cp, 'exec');
-      notificationServiceStub = stub(notificationService, 'showWarningMessage');
-      rootWorkspacePathStub = stub(rootWorkspace, 'getRootWorkspacePath');
+      sandbox = createSandbox();
```

Comment:
Nice on using the sandbox. I know some tests create a sandbox in the beforeEach hook, but i think it really only needs to be made once at the top.